src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer starts working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working");

```

Comment:
Similarly to the `doWork` method, the message 'Bulldozer stopped working' should be more informative. It should clearly indicate what type of `Machine` has stopped working and, if possible, include details about the state after stopping.